src/components/home/HomeChatListByTag/index.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * 사용자 선택 태그 기반 추천 채팅방 리스트
+ * @author 선우
+ */
+
+import { useChatRoomsByTag } from '@/hooks/home/useChatRoomsByTag';
+import profileAtom from '@/recoil/user/profile';
+import { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { SwiperClass, SwiperSlide } from 'swiper/react';
+import 'swiper/swiper-bundle.css';
+import ChatRoomCard from '../ChatRoomCard';
+import PageNavigator from '../PageNavigator';
+import * as Styled from './HomeChatListByTag.styled';
+import { tags } from '@/constants/tag';
+import Tag from '@/components/@common/Tag';
+
+const HomeChatListByTag = () => {
+  // swiper 관련 state
+  const [swiper, setSwiper] = useState<SwiperClass>();
+  const [isBeginning, setIsBeginning] = useState(true);
+  const [isEnd, setIsEnd] = useState(false);
+
+  const [categoryIds, setCategoryIds] = useState<number[]>([]);
+
+  // 전역 변수에 저장된 사용자 정보 가져오기
+  const userInfo = useRecoilValue(profileAtom);
+
+  // 사용자가 선택한 관심사 배열 원소 중, 2개만 추출해서 세팅
+  useEffect(() => {
+    if (userInfo) {
+      const userSelectedTag = userInfo.interestTag
+        .map((tag) => tag.id)
+        .slice(0, 2);
+
+      setCategoryIds([...userSelectedTag]);
```

Comment:
```suggestion
      setCategoryIds(userSelectedTag);
```
userSelectedTag를 바로 저장해도 좋을 것 같은데, 다시 새로운 배열로 만드신 이유가 있을까요...?

Reply:
정신줄 놓고 작성했던 거 같아요 😂 

수정 완료했습니다!